src/components/options/SpcOptionEditor.tsx

```diff
@@ -17,11 +17,19 @@ const aggregationTypeOptions: Array<{ label: string; value: AggregationType }> =
   { label: 'Range', value: 'range' },
   { label: 'Standard deviation', value: 'standardDeviation' },
 ];
+const chartTypeOptions: Array<{ label: string; value: ChartType }> = [
+  { label: 'Mean chart', value: 'meanChart' }, //default
```

Comment:
Instead of `meanChart` use `timeseries`.

- **label** should be set to `Time series`, 
- **value** should be set to `timeseries` 

Please rename all instances of `meanChart` in the code to avoid confusion in the future.

Reply:
done in [cf90bec](https://github.com/KensoBI/spc-panel/pull/48/commits/cf90bec0af252cb0846e58b2ba2280674d256aaf)